config/initializers/okcomputer.rb

```diff
@@ -59,6 +60,16 @@ def check
 
 OkComputer::Registry.register 'ruby_version', OkComputer::RubyVersionCheck.new
 
+# check for backed up resque queues
+Resque.queues.each do |queue|
+  OkComputer::Registry.register "feature-backed-up-#{queue}",
```

Comment:
what do you think of "feature-#{queue}-queue-depth" as a name?

Reply:
Sounds good.